### PR TITLE
Ui 89: update index.js with component urls

### PR DESCRIPTION
**Pull request by andrewtpham**
## What is the Purpose?
Briefly describe what the PR addresses

## What was the approach?
Briefly describe the approach used to address the issue

## Are there any concerns to addressed further before or after merging this PR?
State some additional info if any. For instance running `install` or setting some environment variable(s)

## Mentions?
Mention persons you'd like to review this PR

## Issue(s) affected?
List of issues addressed by this PR.

### CODE REVIEW

**Comment by andrewtpham** [comment]
@michaelbukachi This adds the new components and fixes the import urls. Let's merge this then should be ready for publishing.

File: src/index.js [hunk 1]
```diff
@@ -88,7 +92,7 @@ import VerticalTab from '../components/tab/VerticalTab.vue'
 import WorkspaceTab from '../components/tab/WorkspaceTab.vue'
```
Comment (michaelbukachi) [inline src/index.js:93]: This has been missed.

Comment (andrewtpham) [reply src/index.js:93]: Oh I left that extra `.` since I saw you address it in your upgrade PR 😆

Comment (michaelbukachi) [reply src/index.js:93]: I always replace `..` with `.` otherwise I can't build the library :joy:.

Comment (andrewtpham) [reply src/index.js]: Removed them all ✅

**Pull request merged**